clang/test/CIR/CodeGen/cast.cpp

```diff
@@ -0,0 +1,58 @@
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
```

Comment:
Can you expand this test to cover all the cast types you're adding (or remove any that can't be supported yet)?

Reply:
Has this been addressed?

Reply:
I removed all tests for the currently unsupported casting kinds. I noticed that a few were missing (bool to int/float/ptr) which I readded.

Reply:
It looks like floating casts are still missing. Can you add float-to-double and double-to-float test?